ios/FluentUI/People Picker/ContactCollectionViewCell.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class ContactCollectionViewCell: UICollectionViewCell {
+    static let identifier: String = "ContactCollectionViewCell"
+    var contactView: ContactView
```

Comment:
This should be private, same for identifier.
Throughout your PR, mark methods/constants/properties as private if you they are internal to their class.

Reply:
I agree for the `ContactView`, but not for the identifier. From what I understand, I need to use the identifier in `ContactCollectionView.swift` when registering `ContactCollectionViewCell` using its identifier. I've also check other "cell classes" and the access control for the identifier is `static` or `class` level.